Index: vendor/goetas-webservices/xsd-reader/src/SchemaReader.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/SchemaReader.php	(date 1605198271016)
+++ src/SchemaReader.php	(date 1605198271016)
@@ -38,6 +38,7 @@
 use GoetasWebservices\XML\XSDReader\Schema\Type\SimpleType;
 use GoetasWebservices\XML\XSDReader\Schema\Type\Type;
 use GoetasWebservices\XML\XSDReader\Utils\UrlUtils;
+use Ujamii\OpenImmo\XSDReader\Schema\Type\ComplexTypeMixed;

 class SchemaReader
 {
@@ -524,6 +525,9 @@
         );

         $type = $isSimple ? new ComplexTypeSimpleContent($schema, $node->getAttribute('name')) : new ComplexType($schema, $node->getAttribute('name'));
+        if ($node->hasAttribute('mixed') && $node->getAttribute('mixed')) {
+            $type = new ComplexTypeMixed($schema, $node->getAttribute('name'));
+        }

         $type->setDoc($this->getDocumentation($node));
         if ($node->getAttribute('name')) {
